src/SectorView.cpp

```diff
@@ -645,6 +652,20 @@ void SectorView::SetSelected(const SystemPath &path)
 	UpdateSystemLabels(m_selectedSystemLabels, m_selected);
 }
 
+void SectorView::SwapSelectedHyperspaceTarget()
+{
+	SystemPath tmpTarget = GetHyperspaceTarget();
+	SetHyperspaceTarget(GetSelected());
+	if (m_automaticSystemSelection) {
+		GotoSystem(tmpTarget);
+	} else {
+		RefCountedPtr<StarSystem> system = m_galaxy->GetStarSystem(tmpTarget);
+		SetSelected(system->GetStars()[0]->GetPath());
+	}
+	//GotoSystem(tmpTarget);
```

Comment:
Commented code ? Either use it or remove it ;-)

Or, at least, I'd very much like a comment as to why it needs to be there.